### PR TITLE
Connect offers and transactions in database

### PR DESCRIPTION
**Pull request by sin6pi7**
Adding transaction hash to `forced_trade_offers` is enough - no need to add anything else to other tables.

### CODE REVIEW

File: packages/backend/src/peripherals/database/ForcedTradeOfferRepository.ts [hunk 1]
```diff
@@ -1,108 +1,135 @@
...
-    const rows = await this.knex('forced_trade_offers')
-      .whereNotNull('accepted_at')
-      .select('*')
-
-    return rows.map(acceptedOfferToRecord)
+      .where('id', '=', row.id)
+    return !!updates
   }
 
   async getLatest({
```
Comment (michalsidzej) [inline packages/backend/src/peripherals/database/ForcedTradeOfferRepository.ts:135]: Don't we need this to be the latest but not the accepted offers?

File: packages/backend/src/peripherals/database/BaseRepository.ts [hunk 2]
```diff
@@ -26,6 +26,10 @@ interface DeleteMethod<A extends unknown[]> {
   (...args: A): Promise<number>
 }
 
+interface SaveMethod<T> {
```
Comment (michalsidzej) [inline packages/backend/src/peripherals/database/BaseRepository.ts:29]: Maybe we can call it `UpdateMethod`? What do you think?

Comment (sin6pi7) [reply packages/backend/src/peripherals/database/BaseRepository.ts:29]: update would take a partial - save updates the whole record.

File: packages/backend/src/peripherals/database/ForcedTradeOfferRepository.ts (hunk 1 above)
Comment (sin6pi7) [reply packages/backend/src/peripherals/database/ForcedTradeOfferRepository.ts:135]: This makes more sense, good catch

**Review by michalsidzej — APPROVED** [review]
I like the updated naming, its cleaner

**Pull request merged**